### PR TITLE
Projectile force implementation

### PR DESCRIPTION
**Pull request by meyer28j**
Implemented simple additional force acting on the projectile. Currently adds initial direction and force to projectile initialization and adds a scalar multiple of "propulsion" to the projectile during flight. Still needs tweaking as any effects are only observed in the first 20 or so frames.

### CODE REVIEW

**Review by Daniil-Aleshechkin — APPROVED** [review]

File: src/main/java/com/pss/handlers/GetProjectileForce.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.pss.handlers;
+import com.pss.interfaces.*;
+
+import javax.vecmath.Vector3d;
+
+public class GetProjectileForce implements IGetProjectileForce{
```
Comment (Daniil-Aleshechkin) [inline src/main/java/com/pss/handlers/GetProjectileForce.java]: Can we rename this actually to something less generic like GetConstantContinualProjectileForce?

**Review by Daniil-Aleshechkin — CHANGES_REQUESTED** [review]
Rename the handler

File: src/main/java/com/pss/handlers/GetProjectileForce.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public GetProjectileForce(IGetConfiguration configurationHandler, IGetProjectile projectileHandler) {
+        _configurationHandler = configurationHandler;
+        _projectileHandler = projectileHandler;
+    }
+
+    public Vector3d getProjectileForce() {
+        Vector3d additionalForce = _projectileHandler.getProjectile().getVelocity();
+
+        // TODO: possibly refactor this if-check to occur on initialization instead of every frame
```
Comment (Daniil-Aleshechkin) [inline src/main/java/com/pss/handlers/GetProjectileForce.java]: It's always possible for the velocity of the projectile to be 0, so I think we're fine with your current implementation. I think we can remove this comment

File: src/main/java/com/pss/handlers/GetProjectileForce.java (hunk 1 above)
Comment (meyer28j) [reply src/main/java/com/pss/handlers/GetProjectileForce.java]: Renamed!

File: src/main/java/com/pss/handlers/GetProjectileForce.java (hunk 2 above)
Comment (meyer28j) [reply src/main/java/com/pss/handlers/GetProjectileForce.java]: True true, comment deleted

**Review by Daniil-Aleshechkin — APPROVED** [review]

**Pull request merged**